src/test/resources/dokumentasjon/features/hovedregler/arbeidsforhold/regel_7_1_er_bruker_ansatt_paa_nor_skip.feature

```diff
@@ -85,3 +85,65 @@ Egenskap: Regel 7.1: Er bruker ansatt på et norsk skip?
       | INNENRIKS    |              | NIS            | NOR            | Ja   |
       | INNENRIKS    | INNENRIKS    | NIS            | UTL            | Nei  |
       | UTENRIKS     | INNENRIKS    | NIS            | NIS            | Nei  |
+
+
+  Scenariomal: 7.1 Bruker med godtatt yrkeskode skal få "Ja"
+    Gitt følgende arbeidsforhold fra AAReg
+      | Fra og med dato | Til og med dato | Arbeidsgivertype | Arbeidsforholdstype |
+      | 01.01.2018      |                 | Organisasjon     | MARITIMT            |
+
+    Og følgende arbeidsgiver i arbeidsforholdet
+      | Identifikator | Arbeidsgivertype | Antall ansatte | Konkursstatus |
+      | 1             | BEDR             | 9              |               |
+
+    Og følgende arbeidsavtaler i arbeidsforholdet
+      | Fra og med dato | Til og med dato | Yrkeskode   | Skipsregister | Stillingsprosent |
+      | 01.01.2018      |                 | <Yrkeskode> | NIS           | 100              |
+
+
+    Når regel "7.1" kjøres med følgende parametre
+      | Fra og med dato | Til og med dato | Har hatt arbeid utenfor Norge |
+      | 30.01.2020      | 30.01.2021      | Nei                           |
+
+    Så skal svaret være "<Svar>"
+
+    Eksempler:
+      | Yrkeskode | Svar |
+      | 1         | Nei  |
+      | 3113149   | Ja   |
```

Comment:
Det kan være mer interessant å teste kombinasjonene av ulike verdier på skipsregister, fartsområde og yrkeskode, enn å ta en sjekk på alle yrkeskodene. Foreslår å utvide denne testen til å ta med fartsområde og skipsregister som variabel i tillegg, og å kutte ned på antall yrkeskoder og teste på

Reply:
> Det kan være mer interessant å teste kombinasjonene av ulike verdier på skipsregister, fartsområde og yrkeskode, enn å ta en sjekk på alle yrkeskodene
Det var egentlig mest for å teste om jeg implementerte riktige yrkeskoder. Enig at det er greit med en kombinasjon av de forskjellige variablene

Reply:
Fant forøvrig at det hadde kommet noen feil yrkeskoder i enum klassen utifra disse testene 😅

Reply:
Det er bra, bare å beholde alle yrkeskoder i test hvis du vil 👍